src/test/java/ComputerPlayerTest.java

```diff
@@ -0,0 +1,44 @@
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class ComputerPlayerTest {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2534705897/issues/source?bid=3284885&fileBranchId=3284887#l8)